docs/config/README.md

```diff
@@ -4815,6 +4806,44 @@ By default the module will be shown if any of the following conditions are met:
 format = 'via [V $version](blue bold) '
 ```
 
+## VCS
+
+The `vcs` module displays the current active Version Control System (VCS).
+The module will be shown only if a configured VCS is currently in use.
```

Comment:
As a new starship user, I think it would be helpful here to see some links in this section to the other modules and some links from those modules to this section to help several use cases:
- the user that finds older configurations with explicit modules in the config
- the user that setup their system sometime before this module existed and wants to understand how to upgrade their config

Add docs something like:
> This module replaces the existing fossil, git, hg, and pijul modules that were previously included in $all with a module that finds the first matching VCS by performing the appropriate search for folders (or whatever fossil does differently), in the parent directories.
>
> The configuration of the modules is done with their configuration. See ...